src/datautils/schematicShapeGenerator.ts

```diff
@@ -788,35 +801,18 @@ const createSubkindPerforationTexture = (
   return texture;
 };
 
-/**
- * Yellow gravel
- * @param perforationOptions
- * @returns
- */
-const createSubkindOpenHoleGravelPackTexture = (perforationOptions: PerforationOptions, widestPerfShapeDiameter: number) => {
+const createCanvas = (perfShape: ComplexRopeSegment, options: PerforationOptions): { canvas: HTMLCanvasElement; ctx: CanvasRenderingContext2D } => {
```

Comment:
Seems to be related to perforations, let's change the naming to be more related.